### PR TITLE
257_Add_new_read_method_to_person_DTO

### PR DESCRIPTION
**Pull request by DmytroKhmelovskyi**
Added read single method for person DAL

### CODE REVIEW

**Review by Pain1x — APPROVED** [review]
PUSH ASAP

File: FamilyAccounting.DAL/Repositories/PersonsRepository.cs [hunk 1]
```diff
@@ -70,5 +70,39 @@ public Person Update(int id, Person person)
...
+
+        public Person Get(int id)
+        {
+            var person = new Person();
+            using (var conn = new SqlConnection(connectionString))
+            {
+                var cmd = new SqlCommand();
+                cmd.Connection = conn;
+
+                cmd.CommandText = $"PR_Persons_Read @_Id = {id}";
```
Comment (RomanHulovatyi) [inline FamilyAccounting.DAL/Repositories/PersonsRepository.cs]: Add EXEC

**Review by RomanHulovatyi — CHANGES_REQUESTED** [review]
Add EXEC

File: FamilyAccounting.DAL/Repositories/PersonsRepository.cs (hunk 1 above)
Comment (DmytroKhmelovskyi) [reply FamilyAccounting.DAL/Repositories/PersonsRepository.cs]: Added

**Review by RomanHulovatyi — APPROVED** [review]
GOOD

**Pull request merged**